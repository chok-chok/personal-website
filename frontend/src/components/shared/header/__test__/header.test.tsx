import * as React from 'react';
import Header from '../header';
import { shallow } from 'enzyme';

describe('Test Header component', () => {
  test('Header component renders correctly', () => {
    const tree = shallow(<Header />);
    expect(tree).toMatchSnapshot();
  });
});
