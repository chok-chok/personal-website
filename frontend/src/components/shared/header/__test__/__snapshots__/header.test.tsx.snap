// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Header component Header component renders correctly 1`] = `ShallowWrapper {}`;
